go.mod

```diff
@@ -2,14 +2,6 @@ module github.com/hashicorp/consul
 
 go 1.20
 
-replace (
-	github.com/hashicorp/consul/api => ./api
-	github.com/hashicorp/consul/envoyextensions => ./envoyextensions
-	github.com/hashicorp/consul/proto-public => ./proto-public
-	github.com/hashicorp/consul/sdk => ./sdk
-	github.com/hashicorp/consul/troubleshoot => ./troubleshoot
-)
```

Comment:
Was this removal intentional?

Reply:
I can comment it out instead, I believe `go mod tidy` removed these lines. Thanks.